app/coffeescripts/models/Conference.coffee

```diff
@@ -19,7 +19,7 @@ define [
 
     permissions_data: ->
       has_actions: @get('permissions')['edit'] || @get('permissions')['delete']
-      show_end: @get('permissions')['initiate'] && @get('started_at') && @get('long_running')
+      show_end: @get('permissions')['initiate'] && @get('permissions')['close'] && @get('started_at')
```

Comment:
This check can just be ```@get('permissions')['close']```. That's what the controller checks on the other side of the api call.